template/razor.go

```diff
@@ -228,6 +232,7 @@ func printDebugInfo(r replacement, content string) {
 	if len(matches) > 0 {
 		logString = logString + " \n"
 	}
-	InternalLog.Trace(logString)
-
+	InternalLog.Debug(logString)
```

Comment:
You'll never reach this without being in Trace mode, right? Shouldn't this be Trace then?

Reply:
Question of color :-)